MyString.java

```diff
@@ -3,13 +3,28 @@
  */
 public class MyString {
     public static void main(String args[]) {
-        String hello = "hello";
-        System.out.println(countChar(hello, 'h'));
-        System.out.println(countChar(hello, 'l'));
-        System.out.println(countChar(hello, 'z'));
-        System.out.println(spacedString(hello));
-        //// Put your other tests here.
-    }
+        // String hello = "hello";
+        // System.out.println(countChar(hello, 'h'));
+        // System.out.println(countChar(hello, 'l'));
+
+    //     System.out.println("sap in space -> " + MyString.subsetOf("sap", "space") + " (expected: true)");
+    //     System.out.println("spa in space -> " + MyString.subsetOf("spa", "space") + " (expected: true)");

```

Comment:
There is a lot of effort done in this assignment. Great work!!
There are some slight changes to approaching assignment I want you to start using. 
Talk to me 
Tal